shopfloor_single_product_transfer_mobile/static/src/scenario/single_product_transfer.js

```diff
@@ -239,9 +239,8 @@ const SingleProductTransfer = {
                         this.state.data.quantity = qty;
                     },
                     on_scan: (scanned) => {
-                        const confirmation = this.state.data.asking_confirmation
-                            ? true
-                            : false;
+                        const confirmation =
+                            this.state.data.asking_confirmation || null;
```

Comment:
shouldn't this be like
```suggestion
                            this.state.data.asking_confirmation || "";
```
like you doing it here? https://github.com/OCA/wms/pull/716

Reply:
changed, thanks for your review